common/src/Model/Game.h

```diff
@@ -86,6 +86,8 @@ namespace TrenchBroom {
             TexturePackageType texturePackageType() const;
             void loadTextureCollections(AttributableNode& node, const IO::Path& documentPath, Assets::TextureManager& textureManager, Logger& logger) const;
             bool isTextureCollection(const IO::Path& path) const;
+            std::vector<std::string> fileTextureCollectionExtensions() const;
```

Comment:
You could also return a const reference here, right?

Reply:
Yeah good point, I'll do that

Reply:
Actually it means I would have to do a hack-around in TestGame, this would no longer work:
```
std::vector<std::string> TestGame::doFileTextureCollectionExtensions() const {
    return {};
}
```
Kind of leaks implementation details, so I'll leave it as is.

Reply:
I usually use a static const local variable for these cases, but I'm okay with leaving it, too.